src/state.h

```diff
@@ -96,7 +98,8 @@ typedef struct {
     uint8_t time_act;
     int16_t tr[TR_COUNT];
     int16_t tr_pol[TR_COUNT];
-    int16_t tr_time[TR_COUNT];
+	int16_t tr_time[TR_COUNT];
```

Comment:
indentation off?

Reply:
ugh. fixed what was causing this, so it shouldn't be an issue going forward. I went through my previous commits & pull requests and have hopefully fixed everything now. must've missed this one.

Reply:
looks like indentation is still off here